flake.nix

```diff
@@ -9,31 +9,28 @@
 
   outputs = { self, nixpkgs, libnbtplusplus, ... }:
     let
-      # Generate a user-friendly version number.
+      # User-friendly version number.
       version = builtins.substring 0 8 self.lastModifiedDate;
 
-      # System types to support (qtbase is currently broken for "aarch64-darwin")
+      # Supported systems (qtbase is currently broken for "aarch64-darwin")
       supportedSystems = [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" ];
 
       # Helper function to generate an attrset '{ x86_64-linux = f "x86_64-linux"; ... }'.
       forAllSystems = nixpkgs.lib.genAttrs supportedSystems;
 
-      # Nixpkgs instantiated for supported system types.
+      # Nixpkgs instantiated for supported systems.
       pkgs = forAllSystems (system: nixpkgs.legacyPackages.${system});
-    in
-    {
-      packages = forAllSystems (system: rec {
-        polymc = pkgs.${system}.libsForQt5.callPackage ./nix { inherit version self libnbtplusplus; };
-        polymc-qt6 = pkgs.${system}.qt6Packages.callPackage ./nix { inherit version self libnbtplusplus; };
-        
-        default = polymc;
-      });
 
-      defaultPackage = forAllSystems (system: self.packages.${system}.default);
+      packagesFn = pkgs: rec {
+        default = polymc;
```

Comment:
This means that it would set `nixpkgs.default` if the overlay is being used.

Reply:
Right, I did not think about it. Fixed